### ISSUE TITLE
Accessibility - Individual Goods Screen - Missing Heading - Screen Name

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#42]
The 'Individual goods' screen is missing screen name heading.

**Comment by aquilent-dgemoets** [comment]
See discussion for #20.

**Comment by jfc3-dol** [comment]
Thanks, for correcting this issue. I'm closing this issue.

**Comment by jfc3-dol** [comment]
Oops, pasted wrong comment.

Looks like Android does not read the top banner/header in same way as iOS, which had the text up there as a heading.

Closing issues.
